packages/icons/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 -   Drop support for Internet Explorer 11 ([#31110](https://github.com/WordPress/gutenberg/pull/31110)). Learn more at https://make.wordpress.org/core/2021/04/22/ie-11-support-phase-out-plan/.
 -   Increase the minimum Node.js version to v12 matching Long Term Support releases ([#31270](https://github.com/WordPress/gutenberg/pull/31270)). Learn more at https://nodejs.org/en/about/releases/.
+-   Removed icons: `camera`, `chartLine`, `closeCircleFilled`, `controlsRepeat`, `expand`, as they all have better existing alternatives, and were unused by the block editor.
```

Comment:
Looks like the PR needs a rebase because 3.0.0 was released in the meantime, see https://github.com/WordPress/gutenberg/blob/trunk/packages/icons/CHANGELOG.md#300-2021-05-14.

Reply:
Welp. On it, thanks.

Reply:
Let me know if I this looks good: https://github.com/WordPress/gutenberg/pull/31533/commits/a3316bca13d28a4252a08ba7adf9aaf9a81bccad

Reply:
Cool let me see if I can do that.

Reply:
Took a stab at alternative suggestions in https://github.com/WordPress/gutenberg/pull/31533/commits/22b3d6414353f685d4c9c7fd732b477703bfe9c2.

Reply:
Awesome, thanks!